Makefile

```diff
@@ -258,3 +259,19 @@ check_assets: assets
 		echo "Run 'make assets' and commit the changes to fix the error."; \
 		exit 1; \
 	fi
+
+helm-install:
+	kubectl apply -f tools/helm.yaml
+	helm init --service-account helm --upgrade
+	$(MAKE) upgrade-helm
+
+helm-test: ARGS=--dry-run --debug
```

Comment:
`helm-debug` or `helm-dryrun` would be better since there is an actual `helm test` command that can deploy test pods and verify things.